streampipes-service-base/src/main/java/org/apache/streampipes/service/base/ServiceBaseConfig.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package org.apache.streampipes.service.base;
+
+
+import java.util.Properties;
+
+public class ServiceBaseConfig {
+
+  private static final String ENDPOINT_INCLUDE_KEY = "management.endpoints.web.exposure.include";
```

Comment:
Would it be useful to retrieve these configurations (l27-29) from environment variables and make them configurable?

Reply:
Changes have been made.